### PR TITLE
Unify code to start new backup

### LINKED ISSUE #770: Doesn't perform storage backup on initial backup to flash drive
1. Format a flash drive
2. Go to Settings > System > Backup
Ensure storage backup is enabled by selecting "Backup my files"
Ensure D2D backup is enabled in Expert settings > Device-to-device backups
5. Open "Backup location"
6. Select your formatted flash drive

After the backup initializes it will only perform a device backup. It will not perform a storage backup until the next backup. 

Port of this issue: https://github.com/GrapheneOS/os-issue-tracker/issues/4049

### PR DESCRIPTION
**Pull request by grote**
to ensure that files backup and app backup will always run if they should.

Fixes #770

### CODE REVIEW

File: app/src/main/java/com/stevesoltys/seedvault/worker/BackupRequester.kt [hunk 1]
```diff
@@ -35,6 +41,29 @@ internal class BackupRequester(
     val packageService: PackageService,
 ) : KoinComponent {
 
+    companion object {
+        @UiThread
+        fun requestFilesAndBackup(
```
Comment (t-m-w) [inline app/src/main/java/com/stevesoltys/seedvault/worker/BackupRequester.kt]: Nit, but is this meant to say `requestFilesAndAppBackup` instead? Looks like it's missing a word, maybe not.

**Review by t-m-w — DISMISSED** [review]
Tested working with USB insertion and manual backups.

File: app/src/main/java/com/stevesoltys/seedvault/worker/BackupRequester.kt (hunk 1 above)
Comment (grote) [reply app/src/main/java/com/stevesoltys/seedvault/worker/BackupRequester.kt]: good catch!

**Review by t-m-w — APPROVED** [review]
Typo fix looks good to me

**Pull request merged**